src/components/Footer/Footer.module.css

```diff
@@ -0,0 +1,315 @@
+.footer {
+  background-color: var(--primary-background-color);
+}
+
+.heading {
+  background-color: var(--secondary-background-color);
+  color: var(--light-text-color);
+  font-family: var(--decoraction-font);
+  font-size: 1.25rem;
+  font-weight: 500;
+  padding: 20px;
+  text-align: center;
+  height: 102px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.contact {
+  font-size: 16px;
+  padding: 32px 24px;
+}
+.contactHeading {
+  font-family: var(--decoraction-font);
+  font-size: 1.9375rem;
+}
+
+.contactMail {
+  color: var(--brand-text-color);
+  cursor: pointer;
+  margin-top: 32px;
+  margin-bottom: 59px;
+  transition: 500ms;
+}
+
+.contactMailAdress {
+  font-weight: 600;
+  font-size: 1.25rem;
+  color: var(--brand-text-color);
+}
+
+.contactMailIcon {
+  display: none;
+  fill: var(--dark-brand-color);
+}
+
+.contactMail:hover a,
+.contactMail:focus a {
+  color: var(--light-brand-color);
+}
+
+.contactMail:hover svg,
+.contactMail:focus svg {
+  fill: var(--light-brand-color);
+}
+
+.adressHeading {
+  font-size: 1.25rem;
+  font-weight: 700;
+}
+
+.adressText {
+  margin-top: 12px;
+  font-size: 1.25rem;
+  font-weight: 500;
+}
+
+.adressLine {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  border-bottom: 1px solid #1d1d1d;
+}
+
+.map {
+  border-radius: 12px;
+  margin-top: 64px;
+  width: 100%;
+  height: 233px;
+}
+
+.logo {
+  width: 55%;
+  margin-top: 36px;
+}
+
+.socialList {
+  display: flex;
+  margin-top: 32px;
+  gap: 32px;
```

Comment:
oznacze tylko tu ale margin-top w social i projektPL logo się nie zgadzają dla tabletu oraz tez margin dla sociali w pc

Reply:
Poprawione. Teraz jest ok. W desktopie logo może być trochę inaczej ale to jest rozwiązanie na zasadzie kompromisu, bo sekcja z mapami i adresami jest mocno przekombinowana.